hypothesis-python/docs/changes.rst

```diff
@@ -1930,7 +1930,7 @@ This release further improves our ``.patch``-file support from
 :func:`~hypothesis.strategies.data` (and don't support
 :obj:`@example() <hypothesis.example>`\ ), and various broken edge-cases.
 
-Because :pypi:`libCST` has released version 1.0 which uses the native parser
+Because :pypi:`libCST <libcst>` has released version 1.0 which uses the native parser
```

Comment:
changes in this vein are to silence 302 redirect warnings from linkcheck, and make combing through the remaining output easier